stb/config.py

```diff
@@ -165,10 +165,14 @@ def get_gitlab_api_token() -> None:
 
 @set_app.command("gitlab_api_token")
 def set_gitlab_api_token(
-    token_name: str = typer.Argument(..., help="token name"),
-    token: str = typer.Argument(..., help="token itself"),
+        token_name: str = typer.Option(default=None, prompt="Please enter your gitlab api token name", ),
+        token: str = typer.Option(default=None, prompt="Please enter your token"),
 ) -> None:
     """Set the gitlab api token name for setting up local services"""
+    if token_name is None or token is None:
+        typer.echo("Token name or token is missing. Please, run the command again.")
+        raise typer.Exit()
```

Comment:
Let's exit with status code 1. It's a good practice.

Reply:
I have improved this part to make it more interactive. Now we are providing a user instruction first where he can get the API token, and only after that, we are offering him to enter values. Check values are not None useless, as the typer is not allowing a user to input a blank string, but I have added a check that user input does not contain only whitespaces.